linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/rule.yml

```diff
@@ -69,6 +69,28 @@ references:
     stigid@sle15: SLES-15-010570
     stigid@ubuntu2004: UBTU-20-010074
 
+{{%- if "rhel" in product %}}
+ocil_clause: |-
+    the file integrity application does not exist, or
+    a script file controlling the execution of the file integrity application does not exist, or
+    the file integrity application does not notify designated personnel of changes
```

Comment:
The newlines here translate to a line break (`<br>` tag) in the generated HTML. Line breaks in the middle of sentence look odd.

Reply:
Markdown in connection with the `>-` flow operator would work great here. Now the only way is to use `|-` with long lines - otherwise the `<pre>` section would become extremely ugly.